### PR TITLE
docs: update supported versions

### PR DESCRIPTION
**Pull request by sowjuec**
### Summary

Update `doc/index.md` to reflect the VMware Cloud Foundation 5.0 is not yet supported / tested.

### Type

- [ ] Bugfix
- [ ] Enhancement or Feature
- [ ] Code Style or Formatting
- [X] Documentation
- [ ] Refactoring
- [ ] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

- [ ] Yes, there are breaking changes.
- [X] No, there are no breaking changes.

### Test and Documentation

- [X] Tests have been completed.
- [ ] Documentation has been added or updated.

### Issue References

Closes #000

### CODE REVIEW

**Review by tenthirtyam — APPROVED** [review]
LGTM! 🚀

File: docs/index.md [hunk 1]
```diff
@@ -16,7 +16,6 @@ The following table lists the supported platforms for this module.
 
 Platform                                                     | Support
 -------------------------------------------------------------|------------------------------------
-:fontawesome-solid-cloud: &nbsp; VMware Cloud Foundation 5.0.x on vSAN ReadyNodes | :fontawesome-solid-check:{ .green }
```
Comment (tenthirtyam) [inline docs/index.md:19]: Actually, consider the following:

```markdown
:fontawesome-solid-cloud: &nbsp; VMware Cloud Foundation 5.0.x on vSAN ReadyNodes | :fontawesome-solid-x:{ .red }
```

**Review by tenthirtyam — CHANGES_REQUESTED** [review]
Making a small suggestion.

File: docs/index.md (hunk 1 above)
Comment (sowjuec) [reply docs/index.md:19]: Taken care

**Review by tenthirtyam — APPROVED** [review]
LGTM!

**Pull request merged**